services/migrations/github_exported_data.go

```diff
@@ -0,0 +1,1457 @@
+	Label            string
+	LabelName        string `json:"label_name"`
+	LabelColor       string `json:"label_color"`
+	LabelTextColor   string `json:"label_text_color"`
+	MilestoneTitle   string `json:"milestone_title"`
+	Subject          string
+	TitleWas         string `json:"title_was"`
+	TitleIs          string `json:"title_is"`
+}
+
+// CommentContent returns comment content
+func (g *githubIssueEvent) CommentContent() map[string]interface{} {
+	switch g.Event {
+	case "closed":
+		return map[string]interface{}{}
+	case "head_ref_force_pushed":
+		return map[string]interface{}{}
+	case "moved_columns_in_project":
+		return map[string]interface{}{}
+	case "connected":
+		fields := strings.Split(g.Subject, "/")
+		if len(fields) > 2 {
+			if fields[len(fields)-2] == "issue" {
+				return map[string]interface{}{
+					"type":    "issue_ref",
+					"subject": fields[len(fields)-1],
+				}
+			} else if fields[len(fields)-2] == "pull" {
+				return map[string]interface{}{
+					"type":    "pull_ref",
+					"subject": fields[len(fields)-1],
+				}
+			}
+		}
+		return map[string]interface{}{}
+	case "disconnected":
+		return map[string]interface{}{}
+	case "referenced":
+		tp := "commit_ref"
+		if g.Issue != "" {
+			tp = "issue_ref"
+		} else if g.PullRequest != "" {
+			tp = "pull_ref"
+		}
+		return map[string]interface{}{
+			"type":     tp,
+			"CommitID": g.CommitID,
+		}
+	case "merged":
+		return map[string]interface{}{}
+	case "mentioned":
+		return map[string]interface{}{}
+	case "subscribed":
+		return map[string]interface{}{}
+	case "head_ref_deleted":
+		return map[string]interface{}{}
+	case "head_ref_restored":
+		return map[string]interface{}{}
+	case "milestoned":
+		return map[string]interface{}{
+			"type":           "add",
+			"MilestoneTitle": g.MilestoneTitle,
+		}
+	case "demilestoned":
+		return map[string]interface{}{
+			"type":           "remove",
+			"MilestoneTitle": g.MilestoneTitle,
+		}
+	case "labeled":
+		return map[string]interface{}{
+			"type":           "add",
+			"Label":          g.Label,
+			"LabelName":      g.LabelName,
+			"LabelColor":     g.LabelColor,
+			"LabelTextColor": g.LabelTextColor,
+		}
+	case "renamed":
+		return map[string]interface{}{
+			"OldTitle": g.TitleWas,
+			"NewTitle": g.TitleIs,
+		}
+	case "ready_for_review":
+		return map[string]interface{}{}
+	case "reopened":
+		return map[string]interface{}{}
+	case "unlabeled":
+		return map[string]interface{}{
+			"type":           "remove",
+			"Label":          g.Label,
+			"LabelName":      g.LabelName,
+			"LabelColor":     g.LabelColor,
+			"LabelTextColor": g.LabelTextColor,
+		}
+	case "assigned":
+		return map[string]interface{}{
+			"Actor":   g.Actor,
+			"Subject": g.Subject,
+		}
+	case "added_to_project":
+		return map[string]interface{}{}
+	default:
+		return map[string]interface{}{}
+	}
+}
+
+// CommentStr returns comment type string
+func (g *githubIssueEvent) CommentStr() string {
```

Comment:
As discussed in `#develop` on discord/matrix `review_requested` is currently not covered and will currently result into a empty comment with the content of `{}`. When actor == subject it will result into a self-request review.

```json
  {
    "type": "issue_event",
    "url": "https://github.com/userstyles-world/userstyles.world/pull/97#event-6014856039",
    "pull_request": "https://github.com/userstyles-world/userstyles.world/pull/97",
    "actor": "https://github.com/Gusted",
    "event": "review_requested",
    "subject": "https://github.com/Gusted",
    "created_at": "2022-02-06T10:27:56Z"
  }
  ```

Reply:
Another event not being covered:
```json
  {
    "type": "issue_event",
    "url": "https://github.com/userstyles-world/userstyles.world/pull/57#event-4730692622",
    "pull_request": "https://github.com/userstyles-world/userstyles.world/pull/57",
    "actor": "https://github.com/vednoc",
    "event": "base_ref_changed",
    "title_was": "main",
    "title_is": "develop",
    "created_at": "2021-05-11T19:22:18Z"
  },
  ```
  
![image](https://user-images.githubusercontent.com/25481501/152677458-80353d43-be11-43b3-8b2b-fb367b3ce3b7.png)
![image](https://user-images.githubusercontent.com/25481501/152677461-7788d88f-2274-4898-94f9-7d07b873042c.png)